fabric-biomes-v1/src/main/java/net/fabricmc/fabric/mixin/biome/MixinAddEdgeBiomesLayer.java

```diff
@@ -31,18 +32,18 @@
 import net.fabricmc.fabric.impl.biome.InternalBiomeUtils;
 
 /**
- * Adds edges and shores specified in {@link OverworldBiomes#addEdgeBiome(Biome, Biome, double)} and {@link OverworldBiomes#addShoreBiome(Biome, Biome, double)} to the edges layer.
+ * Adds edges and shores specified in {@link OverworldBiomes#addEdgeBiome(RegistryKey, RegistryKey, double)} and {@link OverworldBiomes#addShoreBiome(RegistryKey, RegistryKey, double)} to the edges layer.
  */
 @Mixin(AddEdgeBiomesLayer.class)
 public class MixinAddEdgeBiomesLayer {
 	@Inject(at = @At("HEAD"), method = "sample", cancellable = true)
 	private void sample(LayerRandomnessSource rand, int north, int east, int south, int west, int center, CallbackInfoReturnable<Integer> info) {
-		Biome centerBiome = BuiltinRegistries.BIOME.get(center);
+		RegistryKey<Biome> centerBiome = Biomes.fromRawId(center);
```

Comment:
I would suggest using `InternalBiomeUtils.fromRawId` similar to `InternalBiomeUtils.getRawId` so we can make them both use a DynamicRegistryManager if needed.

Reply:
Answered above.